docs/sources/experiments/running.md

```diff
@@ -188,6 +188,7 @@ If an experiment fails for any reason, you can restart it.
 
 By default, all work will be recreated. This includes label queries, feature queries, matrix building, model training, etc. However, if you pass the `replace=False` keyword argument, the Experiment will reuse what work it can.
 
+- Cohort Table: The Experiment keeps a cohort table namespaced by its experiment hash, and within that will check on a per-as-of-date level whether or not there are any existing rows, and skip the cohort query for that date if so. For this reason, it is *not* aware of specific entities or source events so if the cohort query has changed or the source data has changed, you will not want to set `replace` to False. Don't expect too much reuse from this, however, as the table is experiment-namespaced. Essentially, this will only reuse data if the same experiment was run prior and failed part of the way through.
```

Comment:
Nitpicky, but for this sentence:

`For this reason, it is *not* aware of specific entities or source events so if the cohort query has changed or the source data has changed, you will not want to set `replace` to False. `

I think we can leave out `if the cohort query has changed` as this will change the experiment hash and not reuse the cohort table.